docs-ref-conceptual/azure-cli-support-lifecycle.md

```diff
@@ -0,0 +1,190 @@
+
+The LTS versions are based on Azure CLI releases that introduce major changes during the second quarter of the calendar year. This usually aligns with Microsoft Build.
+
+LTS releases have the following characteristics:
+
+* During the first six months, the LTS releases are released monthly and receive feature updates, bug fixes and security fixes.
+* During the following 12 months, the LTS releases are released when necessary and only receive bug fixes and security fixes.
+* To be supported, you need to be on the latest version of the LTS release.
+
+Easily find the LTS release date by running the `az version` command. Notice the `azure-cli-support` line that indicates the LTS date.
+
+```output
+{  
+  "azure-cli": "2.48.0",  
+  "azure-cli-core": "2.48.0",  
+  "azure-cli-support": "LTS-20240521",  
+  "azure-cli-telemetry": "1.0.8",  
+  "extensions": {  
+  }  
+}  
+```
+
+The following image illustrates the release cadence and support lifecycle of Azure CLI for STS and LTS releases.
+
+![Support Lifecycle](./media/support-lifecycle/support-lifecycle.png)
+
+The following table provides examples of how the release cadence correlates with the support
+lifecycle:
+
+> [!NOTE]
+> The dates in the following table are for example purposes only. The actual release dates vary.
+
+| Version number | Release schedule | Breaking changes | Release type | Support type | Supported until version |
+| -------------- | ---------------- | ---------------- | ------------ | ------------ | ----------------------- |
+| 2.61.0         | May 2024         | Yes              | Major        | LTS          | 2.62.0                  |
+| 2.62.0         | June 2024        |                  | Minor        | LTS          | 2.63.0                  |
+| 2.63.0         | July 2024        |                  | Minor        | LTS          | 2.64.0                  |
+| 2.64.0         | August 2024      |                  | Minor        | LTS          | 2.65.0                  |
+| 2.65.0         | September 2024   |                  | Minor        | LTS          | 2.66.0                  |
+| 2.66.0         | October 2024     |                  | Minor        | LTS          | _**2.79.0**_            |
+| 2.67.0         | November 2024    | Yes              | Major        | STS          | 2.68.0                  |
+| 2.68.0         | December 2024    |                  | Minor        | STS          | 2.69.0                  |
+| 2.69.0         | January 2025     |                  | Minor        | STS          | 2.70.0                  |
+| 2.70.0         | February 2025    |                  | Minor        | STS          | 2.71.0                  |
+| 2.71.0         | March 2025       |                  | Minor        | STS          | 2.72.0                  |
+| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.90.0**_            |
```

Comment:
My understanding is that as an STS, 2.72.0 support is ended immediately after 2.73.0 is released.

```suggestion
| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.73.0**_            |
```

Reply:
@jiasli , I've made this change in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455), but I feel this needs more conversation. Please leave additional comments in the new PR.